cnf-tests/testsuites/pkg/networkpolicy/multinetworkpolicy.go

```diff
@@ -2,14 +2,33 @@ package networkpolicy
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/onsi/gomega"
+	apiext "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+
 	multinetpolicyv1 "github.com/k8snetworkplumbingwg/multi-networkpolicy/pkg/apis/k8s.cni.cncf.io/v1beta1"
 	client "github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/client"
 )
 
+func IsMultinetPolicyCRDPresentOnCluster() (bool, error) {
```

Comment:
Given the package name, this can be reduced to "IsEnabled()"

Reply:
sure, going to change it